### DISCUSSION TITLE
Scene crashing room so is there a URL option to load a room with a different scene?

### DISCUSSION THREAD

**Discussion by JT-os** [mozilla/hubs#3420] [General Questions & Discussion] [answered]
Is there a way to load a room and change the scene at the same time?  I have a room scene that I was just testing that is not loading on a phone.  So I want to either go back to the previous scene for the room or load a new scene since I can't get past the crash.  

Is there a way to pass a scene code in the URL as an option?

Also, is there a doc page of URL options somewhere?

I know the Hubs bot for Discord will let you create a room with a specific scene as one of its commands it responds to.

**Comment by wmurphyrd** [comment, marked as answer]
If you go to a scene preview from either the Spoke editor or the scenes listing in the Hubs Cloud admin, there's an option to create a room directly with that scene
